docs/control/control_overview.md

```diff
@@ -1,25 +1,38 @@
 
 (open-loop-control)=
 ## Open-loop control framework
-The first approach, open-loop control, assumes no feedback of any kind to the controller. The open-loop framework does not require a detailed performance model and can essentially act as the performance model in the absence of a dedicated performance model for a given storage technology. The open-loop framework establishes a control component that runs the control and passes out information about storage dispatch, state of charge (SOC), etc.
+The first approach, open-loop control, assumes no feedback of any kind to the controller. The open-loop framework does not require a detailed technology performance model and can essentially act as the performance model. The open-loop framework establishes a control component that runs the control and passes out information about `<commodity>_unmet demand`, `unused_<commodity>`, `<commodity>_out`, and `total_<commodity>_unmet_demand`.
+
+Supported controllers:
+- [`pass_through_controller`](#pass-through-controller)
+- [`demand_open_loop_storage_controller`](#demand-open-loop-storage-controller)
+- [`demand_open_loop_converter_controller`](#demand-open-loop-converter-controller)
+- [`flexible_demand_open_loop_converter_controller`](#flexible-demand-open-loop-converter-controller)
+
+(pass-through-controller)=
+### Pass-Through Controller
+The `pass_through_controller` simply directly passes the input commodity flow to the output without any modifications. It is useful for testing, as a placeholder for more complex controllers, and for maintaining consistency between controlled and uncontrolled frameworks as this 'controller' does not alter the system output in any way.
+
+For examples of how to use the `pass_through_controller` open-loop control framework, see the following:
+- `examples/01_onshore_steel_mn`
+- `examples/02_texas_ammonia`
+- `examples/12_ammonia_synloop`
+
+(demand-open-loop-storage-controller)=
+### Demand Open-Loop Storage Controller
+The `demand_open_loop_storage_controller` uses simple logic to dispatch the storage technology when demand is higher than commodity generation and charges the storage technology when the commodity generation exceeds demand, both cases depending on the storage technology's state of charge. For the `demand_open_loop_storage_controller`, the storage state of charge is an estimate in the control logic and is not informed in any way by the storage technology performance model.
```

Comment:
I find it a little confusing to have two structures for controller docs. can we put each in a separate file or all in one file?